docs/CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+
+## 1.0.0b1
+
+Feature:
+* Get configs via Airflow Configuration manager [#503](https://github.com/astronomer/astro-sdk/issues/503)
```

Comment:
We were already using Airflow Configuration manager in a few parts of the code before this ticket. I believe this change is better classified as "Breaking change" than a feature.

Before, users would declare the default Astro SDK temporary schema using: `AIRFLOW__ASTRO__SQL_SCHEMA`, now they should use `AIRFLOW__ASTRO_SDK__SQL_SCHEMA`

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/576/commits/88b0e130c53839808f7fa7be3df528526f57e8ab)